samples/system-test/detect.test.js

```diff
@@ -31,48 +29,43 @@ const audioFilepathBookARoom = path
   .join(__dirname, '../resources/book_a_room.wav')
   .replace(/(\s+)/g, '\\$1');
 
+const exec = cmd => execSync(cmd, {encoding: 'utf8'});
+
 describe('basic detection', () => {
   it('should detect text queries', async () => {
-    const {stdout} = await execa.shell(`${cmd} text -q "hello"`, {cwd});
+    const stdout = await exec(`${cmd} text -q "hello"`);
```

Comment:
Since this method is sync, you don't need to await it!

Reply:
Oops! I removed all the unnecessary awaits and pushed a new commit, thank you!